### DISCUSSION TITLE
[丰饶]药王慈怀，建木生发。莳者一心，同登极乐。

### DISCUSSION THREAD

**Discussion by ShanZheng2333** [share121/inter-knot#4551] [原神]
如是我闻:

一时千手慈怀药王游化诸界，与莳者九万九千九百九十九人俱，住罗喉下，而为说法。

尔时倏忽令使问千手慈怀药王言:「众生有疾，万类皆苦。一切有情众生，缚于生老病苦，何谓苦本?」

千手慈怀药王言:「生老病苦，皆归于死。生者之苦，苦于有涯。老者之苦，苦于必至。病者之苦，苦于近死。是故生老病苦，谓有死。从有死，则为苦本。」
倏忽令使合掌言:「一切有情众生，有灭尽日，有轮回日，何断生老病苦?」

千手慈怀药王告倏忽令使言:「是彼众生，囿于形骸，如囚入笼，受三苦刑。吾今植永寿神木，令彼众生解脱，生无涯，老不至，死回生，断离烦恼。」

尔时倏忽令使作是念言:「药王慈怀，建木生发。莳者一心，同登极乐。」

「药王慈怀，建木生发。莳者一心，同登极乐。」

尔时莳者九万九千九百九十九人俱言:「药王慈怀，建木生发。莳者一心，同登极乐。」

说是语时，莳者九万九千九百九十九人俱，皆断生老病苦。
![微信图片_20240907201619](https://github.com/user-attachments/assets/558a47db-b3e7-406a-b6b5-b5c0b426f1ab)

**Comment by ShanZheng2333** [comment]
药王慈怀，建木生发。莳者一心，同登极乐。

**Comment by Dave-12138** [comment]
十王圣断<img src="https://bbs-static.miyoushe.com/static/2023/12/19/2f3dbb500f3e7f78d9c32f577a155c62_2550392836154570910.png" alt="雪衣 逮捕" width="75" >

**Comment by feertun** [comment]
丰饶孽物，该斩

**Comment by canyusuo** [comment]
药王秘传已经打到原神了吗😱

**Comment by LeeKeeschen** [comment]
终于找到组织了，我一介化外民就是为了寻求长生之法才来的仙舟，请问阁下线下聚会地址在哪

**Comment by love262-wkx** [comment]
大胆丰饶孽物，竟敢在此放肆

**Comment by ll334** [comment]
旅行者厥区零好玩吗

**Comment by xingyuehao** [comment]
药王赐福不过图一乐，千年已是魂灵极限，堕入魔阴万劫不复。<img src="https://img-static.mihoyo.com/communityweb/upload/677e18698f28c655cd39bfb7f16a86f2.png" alt="卡芙卡 怒" width="75" >
真正的长生不老、百病不侵还是要看我们修真者。本仙已经突破渡劫巅峰，达到神变期初期，发现凡人口中的寿瘟祸祖也不过如此，不仅要恪守规则，其力量也无法轻易被凡人接纳。今本仙为造福世人，将自己独创功法散布人间，以此造福世间凡灵。
[点击观看仙人独创功法](https://www.bilibili.com/video/BV1GJ411x7h7/?vd_source=6b8cc71073cf8cdf85991c8776b1c572)

**Reply by ysdtal** [reply]
每天上一当，当当不一样

**Reply by ggggNPC** [reply]
你被骗了

**Reply by love262-wkx** [reply]
我嘞个绳网诈骗啊

**Reply by ID104772508** [reply]
上当+1

**Reply by fqing8485** [reply]
沃日有诈骗

**Reply by Robin-txdy** [reply]
被骗了

**Comment by Bisaqulaor** [comment]
大胆！丰饶孽物！

**Comment by x18144506** [comment]
开门，云骑军

**Comment by Robin-txdy** [comment]
开门，十王司

**Comment by akiirokaede** [comment]
魔阴身，才是仙舟老登唯一的出路！药王慈怀🙏
